build-tools/xa-prep-tasks/Xamarin.Android.BuildTools.PrepTasks/SetEnvironmentVar.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System;
+
+namespace Xamarin.Android.BuildTools.PrepTasks
+{
+	public class SetEnvironmentVar : Task
```

Comment:
We have code completion and (occasional) IDEs. We can use `SetEnvironmentVariable`. :-)